papas/reconstruction/src/PapasManager.cpp

```diff
@@ -51,7 +52,7 @@ void PapasManager::mergeClusters(const std::string& typeAndSubtype) {
   // create collections ready to receive outputs
   auto& mergedClusters = createClusters();
   auto& history = createHistory();
-  MergedClusterBuilder ecalmerger(m_event, typeAndSubtype, ruler, mergedClusters, history);
+  papas::mergeClusters(m_event, typeAndSubtype, ruler, mergedClusters, history);
```

Comment:
Its needed here to distinguish between the PapasManager method and the new mergeClusters function.